Blog: GrowthBook alternatives (#8373)
PostHog/posthog.com: contents/blog/best-growthbook-alternatives.mdx

  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
  <ComparisonRow column1={true} column2={false} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={true} column2={"Edit only"} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={true} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"First party"} column2={"Third party"} feature="Data source" description="Source of data used by flags and A/B tests" />
</ComparisonTable>

### Why do companies use LaunchDarkly?

According to [G2 reviews](https://www.g2.com/products/launchdarkly/reviews#reviews), users appreciate these aspects of LaunchDarkly:

> 1. **SDKs:** Users appreciate how easy it is to integrate LaunchDarkly into their apps thanks to the range of SDKs they provide like JavaScript, Python, Android, and iOS.

Reviewer: ```suggestion
1. **SDKs:** Users appreciate how easy it is to integrate LaunchDarkly into their apps thanks to the range of SDKs it provides like JavaScript, Python, Android, and iOS.
```

> 2. **Automations:** LaunchDarkly provides automations like scheduled rollouts, rollout templates, DevOps pipeline integrations, and stale flag cleanups. Developers mention this as A big selling point.

Reviewer: ```suggestion
2. **Automations:** LaunchDarkly provides automations like scheduled rollouts, rollout templates, DevOps pipeline integrations, and stale flag cleanups. Developers mention this as a big selling point.
```

3. **Speed and availability:** High uptime and speed are critical for developers. Users highlight local caching and edge computing integrations as critical ways LaunchDarkly supports these.

> #### Bottom line

> > LaunchDarkly has an improved feature set to GrowthBook. For enterprises, it is worth considering. For smaller teams, it is unlikely the best option because it lacks self-serve and is closed source.

Reviewer: ```suggestion
> LaunchDarkly has stronger features than GrowthBook. For enterprises, it is worth considering. For smaller teams, it is unlikely the best option because it lacks self-serve and is closed source.
```

## 3. Flagsmith

- **Founded:** 2018
- **Similar to:** Unleash, DevCycle
- **Typical users:** Enterprise engineering teams
- **Typical customers:** Enterprises with strict regulatory requirements

![Flagsmith](https://res.cloudinary.com/dmukukwp6/image/upload/v1713309717/posthog.com/contents/images/blog/best-split-alternatives/flagsmith.png)

### What is Flagsmith?

> Flagsmith is an open source feature flag and [remote configuration](/product-engineers/feature-flags-vs-configuration) service. It enables developers to release and manage features across web, mobile, and server apps. Along with their cloud hosted option, you can host it in your own private cloud or on-premise.

Reviewer: ```suggestion
Flagsmith is an open source feature flag and [remote configuration](/product-engineers/feature-flags-vs-configuration) service. It enables developers to release and manage features across web, mobile, and server apps. Along with a cloud hosted option, you can host Flagsmith in your own private cloud or on-premise.
```

2. **SDKs:** Flagsmith developed a range of SDKs from iOS to Python to JavaScript. This enables users to easily add and use it in their apps.

3. **Integrations:** Users appreciate that Flagsmith integrates with project management, integration pipeline, and analytics tools.

> #### Bottom line
> If you are looking for an open source feature flagging tool, Flagsmith makes a good alternative. It's also good if you have everything but an A/B testing tool setup. If neither is true for you, there are likely better options. 

## 4. VWO

- **Founded:** 2009
- **Similar to:** PostHog, AB Tasty
- **Typical users:** Product managers, engineers, UX designers
- **Typical customers:** Enterprise B2B and B2C companies optimizing customer experiences

![VWO](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/best-launchdarkly-alternatives/vwo.png)

### What is VWO?

[VWO](/blog/best-vwo-alternatives) is a digital experience optimization platform that aims to maximize conversion with tools like A/B testing, personalization, funnels, heatmaps, session replay, and customer analytics.

> The VWO platform is home to multiple different products including testing, insights, data, personalize, plan, and web rollouts each with its own feature sets and pricing.

Reviewer: ```suggestion
The VWO platform is home to a bunch of different products including testing, insights, data, personalize, plan, and web rollouts. Each has its own feature sets and pricing.
```

### Key features

- 🧪 **A/B testing:** Improve and optimize experiences and conversions with web, mobile, and server-side A/B testing.

- 💽 **Data platform:** Collect and analyze customer data across your stack using SDKs, third-party integrations, and direct uploads.

- 🤔 **Insights:** Understand your users with session recordings, heatmaps, analytics, and surveys.

- 👔 **Personalization:** Create and tailor user journeys and campaigns for specific audiences, locations, and times.

- 📋 **Planning:** Ideate, prioritize, and plan optimization campaigns in one location. Connect them directly to data as well.

### How does VWO compare to GrowthBook?

> VWO and GrowthBook have slightly different focuses. Although they both have experimentation, VWO adds personalization and behavioral analytics to their stack. GrowthBook has a greater focus on feature flags. 

Reviewer: ```suggestion
VWO and GrowthBook have slightly different focuses. Although they both have experimentation, VWO adds personalization and behavioral analytics to its stack. GrowthBook has a greater focus on feature flags. 
```

## 5. Unleash

- **Founded:** 2015
- **Similar to:** Flagsmith, DevCycle
- **Typical users:** Regulatory and security conscious developers
- **Typical customers:** Large B2B enterprises

![Unleash](https://res.cloudinary.com/dmukukwp6/image/upload/v1714148743/posthog.com/contents/images/blog/best-split-alternatives/unleash-mega.jpg)

### What is Unleash?

Unleash is a feature management platform that provides an overview of all features across your applications and services. The platform empowers engineering teams to implement A/B tests via feature toggles and offers accurate user targeting.

As of April 2024, Unleash has 10.3k stars on its GitHub repo, nearly double GrowthBook's 5.5k.

### Key features

- ⛳ **Feature flags:** A developer-focused feature flag tool that streamlines the software release process.

- ✅ **Approvals:** Require approvals to make changes to feature flags to ensure compliance.

- 🙋 **Self-hostable:** Choose either fully managed or self-hosted to fit with your needs.

- 🌴 **Environments: Configure flags, targeting, and approvals differently for dev, staging, prod.

### How does Unleash compare to GrowthBook?

> Unleash is very similar to GrowthBook, but doesn't have A/B testing built in. Instead, it provides the feature flag logic and relies on a third party tool, like PostHog, to do the analysis. Other features, including being open source, are nearly identical though.

Reviewer: ```suggestion
Unleash is very similar to GrowthBook, but doesn't have A/B testing built in. Instead, it provides the feature flag logic and relies on a third party tool, like PostHog, to do the analysis. Other features, including being open source, are almost identical.
```

  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={false} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
  <ComparisonRow column1={false} column2={false} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={true} column2={"Edit only"} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={true} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"First party (flags)"} column2={"Third party"} feature="Data source" description="Source of data used by flags and A/B tests" />
</ComparisonTable>

### Why do companies use Unleash?

> According to G2, these are the 

Reviewer: missing the end of this sentence

1. **Self-hostable:** Many users mention they are using the open source version of Unleash and self-hosting. This means teams with compliance concerns can integrate it into their existing infrastructure.

> 2. **Developer-focused:** Unleash focuses on developers. Their tooling and SDKs make it relatively simple for them to implement feature management.

Reviewer: ```suggestion
2. **Developer-focused:** Unleash focuses on developers. Its tooling and SDKs make it relatively simple for them to implement feature management.
```

> 2. **Simple UI:** Even though Unleash is developer-focused, its UI is still praised for making it easy to create and manage toggles. Developers mentioned non-developer are able to set up and manage toggles without help.

Reviewer: ```suggestion
2. **Simple UI:** Even though Unleash is developer-focused, its UI is still praised for making it easy to create and manage toggles. Developers mentioned non-developers are able to set up and manage toggles without help.
```

> #### Bottom line

> > Unleash is very focused on feature flags, and if that is what you are looking for, it makes a good alternative to GrowthBook. If you're searching for A/B testing or other types of optimization and analysis tools, it is best to look elsewhere.

Reviewer: ```suggestion
> Unleash is very focused on feature flags, and if that is what you are looking for, it makes a good alternative to GrowthBook. If you're searching for A/B testing or other types of optimization and analysis tools, it's best to look elsewhere.
```

## 6. DevCycle

- **Founded:** 2022
- **Similar to:** Unleash, Flagsmith
- **Typical users:** Development teams
- **Typical customers:** All stages of B2B software companies

![DevCycle](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/best-launchdarkly-alternatives/devcycle.png)

### What is DevCycle?

> DevCycle is a startup launched out of Taplytics, another A/B testing platform. It is a feature flag management platform built for developers and designed for speed. It has a suite of dev tool integrations, CLI, and simple interface. They use the OpenFeature standard, which enables it to be swapped easily with other providers.

Reviewer: ```suggestion
DevCycle is a startup launched out of Taplytics, another A/B testing platform. It is a feature flag management platform built for developers and designed for speed. It has a suite of dev tool integrations, CLI, and simple interface. It uses the OpenFeature standard, which enables it to be swapped easily with other providers.
```

### Key features

- 🏁 **Feature flags:** Easily and safely rollout and rollback features to deploy faster and reduce risk. Use automation to put an end to manual changes.

- 🤝 **Integrations:** Combine with the tools you are already using for your workflow like GitHub, Terraform, Datadog, and Jira.

- 🧪 **A/B testing:** Run experiments and evaluate the impact of new features. Do multi-armed bandits to compare variations.

- 🧑‍💻 **Developer-focused:** Serverless architecture, edge decisioning support, a CLI, and a robust API help developers make full use of DevCycle.

### How does DevCycle compare to GrowthBook?

> DevCycle has similar upsides and downsides to GrowthBook, but is not open source. They have pulled away from A/B testing as a focus and now mainly focus on feature flagging.

Reviewer: ```suggestion
DevCycle has similar upsides and downsides to GrowthBook, but is not open source. It's pulled away from A/B testing as a focus and now mainly focuses on feature flagging.
```

  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
  <ComparisonRow column1={false} column2={false} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={"Edit only"} column2={"Edit only"} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={"No audit log"} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"First party"} column2={"Third party"} feature="Data source" description="Source of data used by flags and A/B tests" />
</ComparisonTable>

### Why do companies use DevCycle?

According to G2, reviewers appreciate two main areas of DevCycle:

> 1. **Simplicity:** DevCycle makes it easy for reviews to get feature flags set up and start improving their feature management process. Users mention the SDKs and documentation makes implementation easy. 

Reviewer: ```suggestion
1. **Simplicity:** DevCycle makes it easy for people to get feature flags set up and start improving their feature management process. Users mention the SDKs and documentation makes implementation easy. 
```

## 7. AB Tasty

- **Founded:** 2013
- **Similar to:** VWO, LaunchDarkly
- **Typical users:** Marketing and product teams
- **Typical customers:** Large retail and entertainment companies

![AB Tasty](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/best-launchdarkly-alternatives/ab-tasty.png)

### What is AB Tasty?

AB Tasty is a platform for optimizing brand and product experiences. This includes web and feature experimentation, personalization, search, and recommendations.

It helps teams build better end-to-end digital user experiences that drive ROI, engagement, and loyalty. The company focuses on retail, entertainment, and ecommerce companies. According to [BuiltWith](https://trends.builtwith.com/analytics/AB-Tasty), as of April 2024, 2,269 of the top million websites deploy AB Tasty.

### Key features

- 🌐 **Web experimentation:** Run A/B and multivariate tests easily with low/no-code tools like a visual editor and pre-built widgets.

- 🧪 **Feature experimentation:** Test new features on specific users or segments in your server-side or mobile apps.

> - 🫂 **Personalization:** Create personalized experiences with their audience builder and segmentation tools.

Reviewer: ```suggestion
- 🫂 **Personalization:** Create personalized experiences with its audience builder and segmentation tools.
```

  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
  <ComparisonRow column1={false} column2={false} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
  <ComparisonRow column1={"Edit only"} column2={"Edit only"} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
  <ComparisonRow column1={"No audit"} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
  <ComparisonRow column1={"First party"} column2={"Third party"} feature="Data source" description="Source of data used by flags and A/B tests" />
</ComparisonTable>

### Why do companies use AB Tasty?

According to G2 reviews, users choose AB Tasty for the following reasons:

> 1. **Ease-of-use:** Non-technical users can create and manage simple A/B tests using the visual editor. The reports also praise how simple and intuitive the entire platform is.

Reviewer: ```suggestion
1. **Ease-of-use:** Non-technical users can create and manage simple A/B tests using the visual editor. Reviewers also praise how simple and intuitive the platform is.
```

> 2. **Support:** AB Tasty's customer support receives high praise. They even provide an option to help you with recommendations and implementation (for a cost) if you need it.

Reviewer: ```suggestion
2. **Support:** AB Tasty's customer support receives high praise. The company even provides an option to help you with recommendations and implementation (for a cost) if you need it.
```
